### PR TITLE
Fix all mentions of "DamienOS" to "AlphaOS"

### PR DESCRIPTION
**Pull request by CombineSoldier14**

### CODE REVIEW

File: configs/main.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
 			}
 		}
 	},
-	"username": "damienos",
+	"username": "AlphaOS",
```
Comment (AlphaGameDeveloper) [inline configs/main.json]: Usernames should (usually) be lowercase

File: main.py [hunk 2]
```diff
@@ -50,19 +50,19 @@ def boot():
     logger.step("Boot")
     logger.substep("Opening terminal and accept user input")
     logger.info("+----------------------------------------+")
-    logger.info("|             DamienOS build             |")
+    logger.info("|             AlphaOS build             |")
```
Comment (AlphaGameDeveloper) [inline main.py:53]: OMG THE THING IS OFF.  ONE CHARACTER OFF.

**Review by AlphaGameDeveloper — COMMENTED** [review]
A few things can be addressed...

File: main.py (hunk 2 above)
Comment (CombineSoldier14) [reply main.py:53]: It doesn't matter...

**Comment by AlphaGameDeveloper** [comment]
LGTM

**Review by AlphaGameDeveloper — APPROVED** [review]
Looks good to me

**Pull request merged**